databricks/koalas/indexes.py

```diff
@@ -571,6 +571,32 @@ def __getattr__(self, item: str) -> Any:
     def rename(self, name, inplace=False):
         raise NotImplementedError()
 
+    @property
+    def levels(self) -> list:
+        """
+        Names of index columns in list.
+
```

Comment:
We might need a note that this might cause OOM since the result could still be huge?

Reply:
emm, but this is similar to `unique` or `dropduplicates` in a way. And practically, maybe it is used for column index most of the time, so I feel it's okay not to address OOM. Any specific reason to address OOM here? would love to hear some thoughts and learn things!! ^^ @ueshin

Reply:
This is different from `unique` or `drop_duplicates` since they are returning `Series` which is distributed across the cluster so it's safe, whereas this collects data into Driver and returns the lists.

Reply:
Basically we should add a note when:

- collect data into Driver
- use `Window` without partition cols.

Reply:
ahh, thanks for detailed explanation! Appreciate a lot!! @ueshin

Reply:
Could you use `.. note:: ` (e.g., see `to_pandas`)?

Reply:
sure!